task_2.py

```diff
@@ -70,19 +70,34 @@
     type=int,
     description='Interval at which to perform visualization'
 )
+parser.add_argument(
+    '--use-wandb',
+    default=False,
+    type=bool,
+    description='Flag to enable visualization'
+)
 # ------------
 
 # Set random seed
 rand_seed = 1024
 if rand_seed is not None:
     np.random.seed(rand_seed)
+    torch.manual_seed(rand_seed)
 
 # Set output directory
 output_dir = "./"
 if not os.path.exists(output_dir):
     os.makedirs(output_dir)
 
 
+def calculate_map():
+    """
+    Calculate the mAP for classification.
+    """
+    #TODO (Q2.3): Calculate mAP on test set
```

Comment:
We can also write a comment stating they should add the necessary parameters for the function.